lib/moodle_rb/courses.rb

```diff
@@ -69,22 +69,24 @@ def create(params)
       response.parsed_response.first
     end
 
-    def show(id)
+    def show(*id)
+      ids = id.map.with_index do |item, idx|
+        [idx.to_s, item]
+      end.to_h
+
       response = self.class.post(
         '/webservice/rest/server.php',
         {
           :query => query_hash('core_course_get_courses', token),
           :body => {
             :options => {
-              :ids => {
-                '0' => id
-              }
+              id: ids
```

Comment:
This might be my fault as I probably got it wrong in the suggestion, but I've noticed the parameter name here should be `ids`

Reply:
done